web/modules/custom/portfolio_core/portfolio_core.module

```diff
@@ -0,0 +1 @@
+<?php
```

Comment:
The display of <?php in the command line may actually be caused by the lack of a newline (blank line) here. That's also why github highlights this line right a red arrow.